src/SystemLog/Logger.cs

```diff
@@ -70,10 +70,23 @@ void ILogPlugin.Log(string source, LogLevel level, object message)
                     StringBuilder sb = new StringBuilder(source);
                     foreach (char c in GetInvalidFileNameChars())
                         sb.Replace(c, '-');
+
                     var path = Combine(Settings.Default.Path, sb.ToString());
                     Directory.CreateDirectory(path);
                     path = Combine(path, $"{now:yyyy-MM-dd}.log");
-                    File.AppendAllLines(path, new[] { $"[{level}]{log}" });
+
+                    try
+                    {
+                        using (var fs = new FileStream(path, FileMode.OpenOrCreate, FileAccess.ReadWrite, FileShare.Read))
+                        using (var writer = new StreamWriter(fs))
+                        {
+                            writer.WriteLine($"[{level}]{log}");
+                        }
+                    }
+                    catch (IOException)
+                    {
+                        Console.WriteLine("Error writing the log file: " + path);
```

Comment:
@shargon Tested, expction is catched successfully. But without File.AppendAllLines(), this can only record one line.
![image](https://user-images.githubusercontent.com/38493437/81553539-d7f9ec00-93b7-11ea-97ef-b92b5d7e0fe9.png)